mono_repo/lib/src/pub_command.dart

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:args/command_runner.dart';
+import 'package:io/ansi.dart';
+import 'package:io/io.dart';
+
+import 'package:path/path.dart' as p;
+
+import 'utils.dart';
+
+class PubCommand extends Command {
+  @override
+  String get name => 'pub';
+
+  @override
+  String get description => 'Run `pub upgrade` against all packages.';
```

Comment:
Should we add a flag for pub get vs. pub upgrade? Or in general do we also encourage folks to just run pub upgrade?

Reply:
yup